package.json

```diff
@@ -20,7 +20,7 @@
   "scripts": {
     "preversion": "npm test && npm run build",
     "version": "git commit -am \"Update dist for release\" && git checkout -b release && git add -f dist/",
-    "postversion": "git push --tags && git checkout master && git branch -D release && git push",
+    "postpublish": "git push --tags && git push",
```

Comment:
I think this needs to be a little different than Syn since the `version` command creates a release branch and adds the built code.